tests/test_cli_main.py

```
@@ -2022,6 +2150,70 @@ def test_wait_omitted_not_in_extras(self, mock_client):
+class TestValidateTokenCommand:
+    def test_validate_token_valid(self, mock_client):
+        mock_client.validate_token.return_value = {
+            "valid": True,
+            "username": "testuser",
+            "message": "Authenticated as testuser",
+        }
+        result = runner.invoke(app, ["validate-token", "github", "--token", "ghp_test"])  # noqa: S106
+        assert result.exit_code == 0
+        assert "Valid" in result.output
+        mock_client.validate_token.assert_called_once_with(
+            token_type="github",
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "token_type"

(S106)